### PR TITLE
Social: Generate changelog and readme changes for 1.5.0-beta

### PR DESCRIPTION
**Pull request by pablinos**
This updates the plugin and package versions for the 1.5.0-beta release, and updates the changelog and readme files.

#### Other information:

The readme.txt file was out of date with the version on .org, so I've changed the version and added @gmjuhasz as a contributor.

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Does this pull request change what data or activity we track or use?
No,

#### Testing instructions:
Manual checks.

### CODE REVIEW

File: projects/plugins/social/readme.txt [hunk 1]
```diff
@@ -91,25 +91,16 @@ The easiest way is to use the Custom Message option in the publishing options bo
...
 #### Changed
-- Changed the values on the pricing table, and fixed a redirect
-- Move share limits code to the Publicize package
-- Set version to 1.4.0-alpha
-- Social: Aligned Jetpack and Social to use the connection-test-results endpoint in the block editor
-- Updated package dependencies.
-- Updated style for Jetpack Logo icon shown in pre-publish panels for Jetpack and Jetpack Social plugins
-- Update Inspector Panel Jetpack icon color to #1E1E1E
-- Use Jetpack logo in Jetpack Social pre-publish screen for Publicize and Social Preview features
+- Updated package dependencies. [#25993, #26640, #26683, #26705, #26716, #26790, #26791, #26808, #26826, #26829, #26851, #27089]
```
Comment (coder-karen) [inline projects/plugins/social/readme.txt]: I don't think the package dependency updates need to be visible in the readme, but either way you ought to remove the PR numbers:

```suggestion
- Updated package dependencies.
```

**Review by coder-karen — DISMISSED** [review]

File: projects/plugins/social/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
...
+- Integrate the ConnectionError react component to the Social plugin. [#26904]
+- Reshare: Added the reshare UI to the block editor extension [#25993]
+
+### Changed
+- Updated package dependencies. [#25993, #26640, #26683, #26705, #26716, #26790, #26791, #26808, #26826, #26829, #26851, #27089]
+
+### Fixed
+- Social: Fix the connection test endpoint URL [#26892]
+- Social: Fix the path to the connections URL in the editor [#26932]
+
```
Comment (bindlegirl) [inline projects/plugins/social/CHANGELOG.md:20]: Changes that were made on 1.4.2 release branch were never ported back to the trunk so this part of the changelog is missing.
```suggestion

## 1.4.2 - 2022-10-20
### Fixed
- Social: Fix the path to the connections URL in the editor [#26932]

## 1.4.1 - 2022-10-19
### Fixed
- Social: Fix the connection test endpoint URL [#26892]
```

**Review by bindlegirl — CHANGES_REQUESTED** [review]
One small change to keep things clean

**Review by spsiddarthan — DISMISSED** [review]

File: projects/plugins/social/CHANGELOG.md (hunk 2 above)
Comment (pablinos) [reply projects/plugins/social/CHANGELOG.md:20]: Oh no! I'll add that back in. Do you think there could be anything else missing?

**Review by bindlegirl — APPROVED** [review]

**Pull request merged**